VENDORING.md

```diff
@@ -26,13 +26,14 @@ dependency on a package of a specific version or greater up to the next major
 release, without encountering breaking changes.
 
 ## Semantic Versioning
-Annotated version tags should follow Schema Versioning policies.
-According to http://semver.org:
+Annotated version tags should follow [Semantic Versioning](http://semver.org) policies:
 
 "Given a version number MAJOR.MINOR.PATCH, increment the:
-    MAJOR version when you make incompatible API changes,
-    MINOR version when you add functionality in a backwards-compatible manner, and
-    PATCH version when you make backwards-compatible bug fixes.
+
+   1. MAJOR version when you make incompatible API changes,
+   2. MINOR version when you add functionality in a backwards-compatible manner, and
+   3. PATCH version when you make backwards-compatible bug fixes.
```

Comment:
@thaJeztah , the number list is from the http://semver.org/
![sem](https://cloud.githubusercontent.com/assets/20062886/22013789/ca5c4fd0-dcd5-11e6-853e-9eff3588200a.jpg)